android_jni/avifandroidjni/src/main/java/org/aomedia/avif/android/AvifDecoder.java

```diff
@@ -110,11 +110,13 @@ public static boolean decode(ByteBuffer encoded, int length, Bitmap bitmap) {
    *     cores as the thread count. Negative values are invalid. When this value is > 0, it is
    *     simply mapped to the maxThreads parameter in libavif. For more details, see the
    *     documentation for maxThreads variable in avif.h.
+   * @param isScale Is enable image scale in decoding. If true the encoded AVIF image size while be
+   *                decode with size of bitmap.
    * @return true on success and false on failure. A few possible reasons for failure are: 1) Input
    *     was not valid AVIF. 2) Bitmap was not large enough to store the decoded image. 3) Negative
    *     value was passed for the threads parameter.
    */
-  public static native boolean decode(ByteBuffer encoded, int length, Bitmap bitmap, int threads);
+  public static native boolean decode(ByteBuffer encoded, int length, Bitmap bitmap, int threads, boolean isScale);
```

Comment:
We cannot change the behavior of this function alone without updating the behaviors of `nextFrame` and `nthFrame` functions as well.

However, adding scaling support to those functions might involve making a copy of the image in the native code since we may not be able to scale the decoder's copy of the image directly.

If you want this PR to be merged as-is, that's probably ok ,but can you please add a TODO to support scaling on `nthFrame` and the `nextFrame` functions.

Reply:
I talk about removing the `isScal` parameter. As the default function, it will support `nextFrame` and `nthFrame `